go/api/set_commands.go

```diff
@@ -381,4 +381,100 @@ type SetCommands interface {
+	//            For Valkey 8.0 and above, negative cursors are treated like the initial cursor("0").
+	//
+	// Return value:
+	//  An array of the cursor and the subset of the set held by `key`. The first element is always the `cursor` and
+	//  for the next iteration of results. The `cursor` will be `"0"` on the last iteration of the set.
+	//  The second element is always an array of the subset of the set held in `key`.
+	//
+	// Example:
+	//	 // assume "key" contains a set
+	// 	 resCursor, resCol, err := client.sscan("key", "0")
+	//   for resCursor != "0" {
+	// 	 	resCursor, resCol, err = client.sscan("key", "0")
+	//   	fmt.Println("Cursor: ", resCursor.Value())
+	//   	fmt.Println("Members: ", resCol.Value())
+	//   }
+	//   // Output:
+	// 	 // Cursor:  48
+	//   // Members:  ['3', '118', '120', '86', '76', '13', '61', '111', '55', '45']
+	//   // Cursor:  24
+	//   // Members:  ['38', '109', '11', '119', '34', '24', '40', '57', '20', '17']
+	//   // Cursor:  0
+	//   // Members:  ['47', '122', '1', '53', '10', '14', '80']
+	//
+	// [valkey.io]: https://valkey.io/commands/sscan/
+	SScan(key string, cursor string) (Result[string], Result[[]string], error)
+
+	// Iterates incrementally over a set.
+	//
+	// Note: When in cluster mode, all keys must map to the same hash slot.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//   key - The key of the set.
+	//   cursor - The cursor that points to the next iteration of results.
+	//            A value of `"0"` indicates the start of the search.
+	//            For Valkey 8.0 and above, negative cursors are treated like the initial cursor("0").
+	//   options - [BaseScanOptions]
+	//
+	// Return value:
+	//  An array of the cursor and the subset of the set held by `key`. The first element is always the `cursor` and
+	//  for the next iteration of results. The `cursor` will be `"0"` on the last iteration of the set.
+	//  The second element is always an array of the subset of the set held in `key`.
+	//
+	// Example:
+	//	 // assume "key" contains a set
+	//   resCursor resCol, err := client.sscan("key", "0", opts)
+	//   for resCursor != "0" {
+	//   	opts := api.NewBaseScanOptionsBuilder().SetMatch("*")
+	// 	 	resCursor, resCol, err = client.sscan("key", "0", opts)
+	//   	fmt.Println("Cursor: ", resCursor.Value())
+	//   	fmt.Println("Members: ", resCol.Value())
+	//   }
+	//   // Output:
+	// 	 // Cursor:  48
+	//   // Members:  ['3', '118', '120', '86', '76', '13', '61', '111', '55', '45']
+	//   // Cursor:  24
+	//   // Members:  ['38', '109', '11', '119', '34', '24', '40', '57', '20', '17']
+	//   // Cursor:  0
+	//   // Members:  ['47', '122', '1', '53', '10', '14', '80']
+	//
+	// [valkey.io]: https://valkey.io/commands/sscan/
+	SScanWithOptions(key string, cursor string, options *BaseScanOptions) (Result[string], Result[[]string], error)
+
+	// Moves `member` from the set at `source` to the set at `destination`, removing it from the source set.
```

Comment:
Please create a task to discuss and align doc format - somewhere function name is included:
> SMove moves ....